src/modules/mavlink/mavlink_mission.cpp

```diff
@@ -288,6 +328,51 @@ MavlinkMissionManager::send_mission_current(int32_t seq)
 	}
 }
 
+
+uint32_t crc32combine(uint32_t crc_a, uint32_t crc_b, size_t len_b)
+{
+	size_t i;
+	uint8_t zero = 0;
+
+	for (i = 0;  i < len_b;  i++) {
+		crc_a = crc32part(&zero, 1, crc_a);
+	}
+
+	return (crc_a ^ crc_b);
+}
```

Comment:
Where does this come from? Is it specified somewhere? And how does it work? I don't understand why you would add the CRC32 over a buffer of one 0 byte.

Reply:
That works because CRC32 is linear, there is an explanation here:
https://stackoverflow.com/questions/23122312/crc-calculation-of-a-mostly-static-data-stream

Reply:
I see. Can you add that as a comment to the file for a reader?

Reply:
Done